config/environments/test.rb

```diff
@@ -45,7 +45,7 @@
   config.action_mailer.default_url_options = { host: 'localhost', port: '3000' }
 
   # Randomize the order test cases are executed.
-  config.active_support.test_order = :random
+  config.active_support.test_order = :sorted
```

Comment:
was this a necessary change?

Reply:
No.   But it makes it easier to do diffs on different versions of the Rspec output. With random, it isn't simple to do this.   I can change it back if you feel that is better.

Reply:
I think it makes sense to leave it as you have it while working through all the upgrades. As long as we change it back once we have everything upgraded and working that should be good.

Reply:
That sounds right. Thanks.